pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -1188,4 +1201,164 @@ private void checkRequestTimeout() {
     }
 
     private static final Logger log = LoggerFactory.getLogger(ClientCnx.class);
+
+    /**
+     * Check if the client connection is eligible for being marked as idle. 
+     *  if so, {@link org.apache.pulsar.client.impl.ClientCnx#tryMarkIdle()} will be invoked next.
+     * @return if the connection is idle.
+     */
```

Comment:
```suggestion
    /**
     * Check if the client connection is eligible for being marked as idle.
     *  if so, {@link org.apache.pulsar.client.impl.ClientCnx#tryMarkIdle()} will be invoked next.
     *  @return true if the connection is eligible 
     */
```

Reply:
> *  @return true if the connection is eligible 

Hi @gaozhangmin 
This line ends with `space`, style check will fail.